gaia-button.js

```diff
@@ -24,6 +24,28 @@ module.exports = component.register('gaia-button', {
     this.setupShadowRoot();
     this.setAttribute('role', 'button');
     this.tabIndex = 0;
+    this.addEventListener('keydown', this._handleKeyEvent.bind(this));
+    this.addEventListener('keyup', this._handleKeyEvent.bind(this));
+    this.addEventListener('blur', this._handleKeyEvent.bind(this));
+  },
+
+  /**
+   * Handle 'Return' key events. Other key events are ignored.
+   */
+  _handleKeyEvent: function(e) {
+    // If the keystroke is not the 'Return' key
+    if (e.keyCode !== KeyEvent.DOM_VK_RETURN) {
+      return;
+    }
+
+    if (e.type === 'keydown') {
+      this.toggled = true;
+    } else if (e.type === 'keyup') {
+      this.toggled = false;
+      // 'released' attribute is for animation
+      this.setAttribute('released', '');
```

Comment:
I see no CSS selectors that use `[released]`. Can you point me to where this is. Also why use an attribute and not a class?